roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -18,7 +18,7 @@ class LoggingCheck(OpenShiftCheck):
     logging_namespace = "logging"
 
     def is_active(self):
-        logging_deployed = self.get_var("openshift_hosted_logging_deploy", default=False)
+        logging_deployed = self.get_var("openshift_hosted_logging_deploy", convert=bool, default=False)
```

Comment:
I don't think I filed a bug. But yeah, it's a minor bug that logging is considered always on if defined...